src/devices/Mlx90640/Mlx90640.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp2.1</TargetFrameworks>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Mlx90640.cs" />
+    <Compile Include="Sampling.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="category.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="README.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" Version="5.0.0" />
+    <PackageReference Include="System.Drawing.Primitives" Version="4.3.0" />
```

Comment:
do you need to include this directly? If so please add the version to https://github.com/dotnet/iot/blob/master/eng/Versions.props and use the variable